frontend/src/metabase/public/components/EmbedFrame/LogoBadge/LogoBadge.module.css

```diff
@@ -1,5 +1,4 @@
 .metabaseLink {
-  --mb-color-text-primary: var(--mb-color-text-medium);
```

Comment:
Remove the declaration, so `.dark` can inherit `--mb-color-text-primary` while being able to override it for light theme to match the design.